site/content/pages/organizing/01-Getting started/03-website-and-email.txt

```diff
@@ -0,0 +1,19 @@
+---
+extension: html
+filter:
+- erb
+- markdown
+title: Website and Email
+---
+When you have your initial team, [email the global core organizers](mailto:info@devopsdays.org) and we'll set up the @devopsdays.org mail aliases for your team. This will let you have a standard email for organizer contact and proposals. 
+
+The other thing you'll want to do as soon as possible is get your event listed on the website by submitting a pull request to [https://github.com/jedi4ever/devopsdays-webby](https://github.com/jedi4ever/devopsdays-webby). The simplest way to do this:
+
```

Comment:
In the past we've encouraged people to wait until they are ready to open registration, accept talks, etc. I've been deliberately moving this towards "show your work even if it's not finished".